src/calc-length.js

```diff
@@ -69,9 +69,9 @@
     var calcDictionary = {};
 
     // Iterate through all length types and multiply all non null lengths 
-    for(var i = 0; i < shared.LengthValue.LengthType.length; i++){
+    for (var i = 0; i < shared.LengthValue.LengthType.length; i++){
```

Comment:
Missing space between bracket and brace here. Also on line 74, 86, 88, 104, 106, 126, 130, 145, 150, 152.